util/cache/twolevelclient.go

```diff
@@ -0,0 +1,52 @@
+package cache
+
+import (
+	"context"
+	"time"
+)
+
+func NewTwoLevelClient(client CacheClient, inMemoryExpiration time.Duration) *twoLevelClient {
+	return &twoLevelClient{inMemoryCache: NewInMemoryCache(inMemoryExpiration), client: client}
+}
+
+type twoLevelClient struct {
+	inMemoryCache *InMemoryCache
+	client        CacheClient
```

Comment:
I think it would be great to have some consistency in the member names, i.e. `inMemoryCache` and `externalCache`. But this is just nitpicking.

Reply:
👍  Renamed